magda-web-client/src/Components/Common/DataPreviewMapOpenInNationalMapButton.tsx

```diff
@@ -13,7 +14,7 @@ type PropsType = {
     };
 };
 
-const DEFAULT_TARGET_URL = "https://nationalmap.gov.au/";
```

Comment:
let's recover this change and test the site again.
I think pete is back today and he should be able to give you access to the cluster.
If you still can't access it, please let me know and I can't modify the config map.
By the way, can I confirm the changes of this PR is depends on this PR?
https://github.com/TerriaJS/terriajs/pull/6093
What will happen for a magda site that applied this PR and send request a terria map site that hasn't applied  PR https://github.com/TerriaJS/terriajs/pull/6093?

Reply:
- `const DEFAULT_TARGET_URL = "https://nationalmap.gov.au/";` has been restored.
- TerriaJS/terriajs#6093 is a dependency. A magda site should not be applied with this PR until the target site has been applied with TerriaJS/terriajs#6093.
- If a magda site is not applied with this PR but the target site has been applied with TerriaJS/terriajs#6093, there will be no effect but I will double check for this.

Reply:
Thanks 👍 I think to go for the safer option, we probably can merge it after it's deployed to national map & NSW digital twin

Reply:
I've tested it locally. If a target site has been applied with TerriaJS/terriajs#6093 but the magda site has not been applied with this PR. There will be no effect, that is, most many wms format data can not be properly opened in the target site.

Reply:
For those wms datasets that is currently working it will still work, right?